cypress/e2e/auddio-API-tests.cy.js

```diff
@@ -0,0 +1,151 @@
+describe('happy-path', () => {
+  describe('intial setup of app when loaded', () => {
+    beforeEach(() => {
+      cy.visit('http://localhost:3000')
+    })
+
+    it('text in button should change to recording once clicked', () => {
+      cy.get('[data-automation="record-button"]').click()
+      cy.get('[data-automation="record-button"]').contains('Recording...')
+    })
+
+    it('should not show any cards or menu before recording button is clicked', () => {
+      cy.get('[data-automation="songCard"]').should('not.exist')
+      cy.get('[data-automation="songNotFoundCard"]').should('not.exist')
+      cy.get('[data-automation="menuBar"]').should('not.exist')
+    })
+
+    it('after 10 seconds text should change back', () => {
+      cy.clock()
+      cy.get('[data-automation="record-button"]').click()
+      cy.get('[data-automation="record-button"]').contains('Recording...')
+      cy.tick(11000)
+      cy.get('[data-automation="record-button"]').contains('Click')
+    })
+
+    it('record button should be disabled after one click', () => {
+      cy.get('[data-automation="record-button"]').click()
+      cy.get('[data-automation="record-button"]').should('be.disabled')
+    })
+    it('record button should be reenabled after 10 seconds of being disabled', () => {
+      cy.clock()
+      cy.get('[data-automation="record-button"]').click()
+      cy.get('[data-automation="record-button"]').should('be.disabled')
+      cy.tick(11000)
+      cy.get('[data-automation="record-button"]').should('not.be.disabled')
+    })
+
+    it('should return songNotFound if result is equal to null and menuBar should not be visible', () => {
```

Comment:
This test could live in it's own describe as it's specific to the scenario where song not found.

Reply:
added song not found describe